db-models/comments.sql

```diff
@@ -0,0 +1,9 @@
+-- db-models/comments.sql
+
+CREATE TABLE comments (
+    comment_id SERIAL PRIMARY KEY,          -- Unique for each comment
```

Comment:
Two things here:
1. The `SERIAL` keyword does not exist in SQLite, we'll need to use INTEGER
2. We'll also need to specify `AUTOINCREMENT` so that SQLite knows to automatically increment each row's integer to 2, to 3, to 4, etc.

Reply:
Nice catch! I'm on it!